src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.CurlResponseMessage.cs

```diff
@@ -426,10 +431,25 @@ private void CheckDisposed()
                 }
             }
 
-            [Conditional(VerboseDebuggingConditional)]
-            private void VerboseTrace(string text = null, [CallerMemberName] string memberName = null)
+            private void EventSourceTrace<T>(string formatMessage, T arg0, [CallerMemberName] string memberName = null)
```

Comment:
Yes, it's important they're generic.  Otherwise the call sites, which mostly pass value types as the arguments, will end up boxing.  We could make them non-generic and have them take objects and even a params array, but then we'll be forced to surround every call site with ```if (EventSourceTracingEnabled)``` to prevent those allocations from occurring when tracing is disabled.